apipie-bindings.gemspec

```diff
@@ -22,17 +22,15 @@ EOF
   s.extra_rdoc_files = `git ls-files -- {doc,config}/* README*`.split("\n")
   s.require_paths    = ["lib"]
 
-  s.add_dependency 'json', '>= 1.2.1'
-  s.add_dependency 'rest-client', '>= 1.6.5', '< 3.0.0'      # lower versions don't allow setting infinite timeouts, higher versions have different api
-  s.add_dependency 'oauth'
+  # lower versions don't allow setting infinite timeouts, higher versions have different api
```

Comment:
I think that comment really was for `< 1.6.5`. Let's ignore that. And pinning to `~> 2.0` implicitly says "we respect semver, next major might have a different API".
```suggestion
```